utils/variables.py

```diff
@@ -6,7 +6,8 @@
 """
 Hyper-parameters that can be changed
 """
-device = 'cuda' if torch.cuda.is_available() else 'cpu'
+# device = 'cuda' if torch.cuda.is_available() else 'cpu'

```

Comment:
our code is not CUDA ready. Need to further fix types. @ChayutWo @youngkyungkim

Reply:
You don't need to remove this line even if you don't have cuda. It will just use your cpu. The only reason why this may cause bug is that your gpu driver is not updated.